testing-common/integration-tests/src/test/groovy/AgentInstrumentationSpecificationTest.groovy

```diff
@@ -17,12 +17,19 @@ import java.util.concurrent.TimeoutException
 class AgentInstrumentationSpecificationTest extends AgentInstrumentationSpecification {
   private static final ClassLoader BOOTSTRAP_CLASSLOADER = null
 
+  /**
+   * Copy of {@link Constants#BOOTSTRAP_PACKAGE_PREFIXES} because the Constants class
+   * is not accessible from here
+   */
+  public static final List<String> BOOTSTRAP_PACKAGE_PREFIXES =
```

Comment:
This test seemed to have relied on the side effect of `Constants` being injected into every ClassLoader containing ClassLoaders. We don't inject it anymore, which caused this test to fail for indy.

Reply:
I think instead of copying it you could add `testCompileOnly(project(":javaagent-bootstrap"))` and use `BootstrapPackagePrefixesHolder.getBoostrapPackagePrefixes()`

Reply:
Definitely better than copying, fixed in https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/12242/commits/57d73cd813fbc79f5d7304a6ad71cf3f8ba20609